### PR TITLE
Expand description/add missing Bio-Formats repository jobs

### PR DESCRIPTION
**Pull request by sbesson**
This PR should complete the knowledge of the current Bio-Formats jobs by:
- adding the Windows jobs
- adding the repository-subset jobs
- adding more details on the breaking jobs

/cc @bramalingam @mtbc @melissalinkert 

--no-rebase

### CODE REVIEW

File: contributing/ci-bio-formats.txt [hunk 1]
```diff
@@ -40,19 +40,25 @@ All jobs are listed under the :jenkinsview:`Bio-Formats` view tab of Jenkins.
...
-		* 
+		* | :term:`BIOFORMATS-5.1-merge-full-repository`
+		  | :term:`BIOFORMATS-5.1-merge-full-repository-win`
+		*
 
 	-	* Runs automated tests against test_images_good on squig
-		* :term:`BIOFORMATS-5.1-merge-test_images_good`
-		* 
+		* | :term:`BIOFORMATS-5.1-merge-test_images_good`
+		  | :term:`BIOFORMATS-5.1-merge-test_images_good`
```
Comment (mtbc) [inline contributing/ci-bio-formats.txt]: this is meant to be a duplicate?

File: contributing/ci-bio-formats.txt [hunk 2]
```diff
@@ -126,27 +132,56 @@ The branch for the 5.1.x series of Bio-Formats is develop.
...
+		   :file:`\\squig.openmicroscopy.org.uk\ome-data-repo\test_images_good`
+
+	:jenkinsjob:`BIOFORMATS-5.1-merge-repository-subset`
+
+		This job runs automated tests against a subset of the data repository
+
+		#. |merge|
+		#. Run automated tests against a subset of format directories under
+		   :file:`/ome/data_repo/from_skyking/`. The list of directories to
+		   test by setting a space separated list of formats for the
```
Comment (mtbc) [inline contributing/ci-bio-formats.txt]: space-separated

File: contributing/ci-bio-formats.txt [hunk 3]
```diff
@@ -165,11 +200,76 @@ breaking jobs are listed under the :jenkinsview:`Breaking` view tab of
...
+		   :term:`BIOFORMATS-5.1-breaking-test_images_good` and
+		   :term:`BIOFORMATS-5.1-breaking-test_images_good-win`
+
+		See :jenkinsjob:`the build graph <BIOFORMATS-5.1-breaking-trigger/lastSuccessfulBuild/BuildGraph>`
 
+	:jenkinsjob:`BIOFORMATS-5.1-breaking-push`
+
+		This job merges all the breaking PRs opened against develop
+
+		#. |merge| labelled as breaking
```
Comment (mtbc) [inline contributing/ci-bio-formats.txt]: "labeled" is more usual for American spelling

File: contributing/ci-bio-formats.txt (hunk 1 above)
Comment (hflynn) [inline contributing/ci-bio-formats.txt]: @sbesson ?

**Comment by sbesson** [comment]
Pushed

File: contributing/ci-bio-formats.txt [hunk 4]
```diff
@@ -126,27 +132,56 @@ The branch for the 5.1.x series of Bio-Formats is develop.
...
 		#. Runs automated tests against :file:`/ome/data_repo/from_skyking/`
 
+	:jenkinsjob:`BIOFORMATS-5.1-merge-full-repository-win`
+
+		This job runs automated tests against the full repository on Windows
+
+		#. Checks out the develop/merge/daily branch of the
+		   snoopycrimecop fork of bioformats.git_
+		#. Runs automated tests against
+		   :file:`\\squig.openmicroscopy.org.uk\ome-data-repo\from_skyking`
```
Comment (hflynn) [inline contributing/ci-bio-formats.txt]: slashes either side of ome-data-repo aren't being rendered, they need to be escaped. Also, you have /ome/data_repo/from_skyking/ above which I assume is referring to the same thing, so inconsistency here

File: contributing/ci-bio-formats.txt [hunk 5]
```diff
@@ -126,27 +132,56 @@ The branch for the 5.1.x series of Bio-Formats is develop.
...
+
+	:jenkinsjob:`BIOFORMATS-5.1-merge-test_images_good-win`
+
+		This job runs automated tests against the test_images_good subset on
+		Windows
+
+		#. Checks out the develop/merge/daily branch of the
+		   snoopycrimecop fork of bioformats.git_
+		#. Runs automated tests against
+		   :file:`\\squig.openmicroscopy.org.uk\ome-data-repo\test_images_good`
```
Comment (hflynn) [inline contributing/ci-bio-formats.txt]: As above, slashes around ome-data-repo are not rendered on staging

File: contributing/ci-bio-formats.txt [hunk 6]
```diff
@@ -165,11 +200,76 @@ breaking jobs are listed under the :jenkinsview:`Breaking` view tab of
...
+
+	:jenkinsjob:`BIOFORMATS-5.1-breaking-test_images_good-win`
+
+		This job runs automated tests against the test_images_good subset on
+		Windows
+
+		#. Checks out the develop/breaking/trigger of the
+		   snoopycrimecop fork of bioformats.git_
+		#. Runs automated tests against
+		   :file:`\\squig.openmicroscopy.org.uk\ome-data-repo\test_images_good`
```
Comment (hflynn) [inline contributing/ci-bio-formats.txt]: As above, slashes around ome-data-repo not rendered on staging and inconsistency in how you refer to this folder

File: contributing/ci-bio-formats.txt (hunk 4 above)
Comment (sbesson) [inline contributing/ci-bio-formats.txt]: Will fix the escaping. The name is actually right, `/ome/data_repo/` is mounted as SMB share `ome-data-repo` for the Windows testing.

**Comment by hflynn** [comment]
Sorry, found an issue with your references to the ome data repo images folders when I doublechecked this on staging

**Comment by sbesson** [comment]
Pushed

**Comment by hflynn** [comment]
Thanks!

**Pull request merged**